issue found on: storage/storage.go:107:1: paramTypeCombine

an issue has been detected in source code: ./storage/storage.go:107:1: paramTypeCombine: func(path string, clusterName string) (string, error) could be replaced with func(path, clusterName string) (string, error)

--- a/storage/storage.go
+++ b/storage/storage.go
@@ -104,7 +104,7 @@ const changingClustersPeriodInMinutes = 15
 
 var reports map[string]string = make(map[string]string)
 
-func readReport(path string, clusterName string) (string, error) {
+func readReport(path, clusterName string) (string, error) {
 	absPath, err := filepath.Abs(path + "/report_" + clusterName + ".json")
 	if err != nil {
 		return "", err

